modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -289,8 +290,7 @@ public void processOpts() {
 
         if (additionalProperties.containsKey(ADDITIONAL_MODEL_TYPE_ANNOTATIONS)) {
             String additionalAnnotationsList = additionalProperties.get(ADDITIONAL_MODEL_TYPE_ANNOTATIONS).toString();
-
-            this.setAdditionalModelTypeAnnotations(Arrays.asList(additionalAnnotationsList.split(";")));
+            this.setAdditionalModelTypeAnnotations(Arrays.asList(additionalAnnotationsList.split("\\s*(;|\\r?\\n)\\s*")));
```

Comment:
`Arrays.asList` use set to remove duplicates

Reply:
I didn't want to change the original code that List uses.
Do you think this is "necessary"?

Reply:
Using stream and distint to avoid duplicates and potentiel side effect of using Set

Reply:
From performance perspective stream.distinct is many times slower than loading things to hashset.

Reply:
So something like String[] -> Set -> List
```java
new ArrayList<>(Sets.newHashSet(additionalAnnotationsList.trim().split("\\s*(;|\\r?\\n)\\s*")))
```